mllib-local/src/main/scala/org/apache/spark/ml/linalg/Matrices.scala

```diff
@@ -1076,4 +1240,15 @@ object Matrices {
       SparseMatrix.fromCOO(numRows, numCols, entries)
     }
   }
+
+  private[ml] def getSparseSize(numActives: Long, numPtrs: Long): Long = {
+    // 8 * values.length + 4 * rowIndices.length + 4 * colPtrs.length + 8 + 8 + 1
+    12L * numActives + 4L * numPtrs + 17L
```

Comment:
I wondered how confusing this comment might be. Since `values.length == rowIndices.length == numActives`:

`8 * values.length + 4 * rowIndices.length = 8 * numActives + 4 * numActives = 12 * numActives`

The comment is meant to show where each number comes from and the implementation is meant to just be a condensed computation. But please let me know if you think it's too confusing.